e2e/cypress/support/extended_commands.js

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import * as TIMEOUTS from '../fixtures/timeouts';
+
+Cypress.Commands.add('visitAndWait', (url, options, duration = TIMEOUTS.THREE_SEC) => {
+    cy.visit(url, options).wait(duration);
```

Comment:
Maybe instead of explicit wait, we can try this to see if page is loaded.
```
            cy.waitUntil(() => cy.get('html').then((el) => {
                return el.find('#root > div > div').length > 0;
            }));
```

Reply:
I've tried this and I still see the `Loading` indicators in channel then post list view remained longer than usual. Let's have as is but this is well noted. We can change anytime in the future.